src/Controls/src/Core/HandlerImpl/Label/Label.Tizen.cs

```diff
@@ -7,12 +7,12 @@ namespace Microsoft.Maui.Controls
 {
 	public partial class Label
 	{
-		public static void MapTextType(LabelHandler handler, Label label) => MapText((ILabelHandler)handler, label);
+		public static void MapTextType(LabelHandler handler, Label label) => MapTextType((ILabelHandler)handler, label);
 		public static void MapText(LabelHandler handler, Label label) => MapText((ILabelHandler)handler, label);
 
 		public static void MapTextType(ILabelHandler handler, Label label)
 		{
-			Platform.TextExtensions.UpdateText(handler.PlatformView, label);
+			handler.UpdateValue(nameof(ILabel.Text));
```

Comment:
Instead of calling the same method, we should be triggering the Text property so people can properly intercept it.